packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -187,67 +199,70 @@ Previous
       <div className={cx(['paginator', !this.hasNavContext() && 'pageless'])} role="navigation" aria-label="pagination">
         {
           this.hasNavContext() && (
-          <a
+          <button
             aria-disabled={selectedPage === 1}
-            aria-label="first"
+            aria-label={intl.formatMessage({ id: 'Terra.paginator.first' })}
             className={cx(['nav-link', 'left-controls', selectedPage === 1 && 'is-disabled'])}
             tabIndex={selectedPage === 1 ? null : '0'}
             onClick={this.handlePageChange(1)}
             onKeyDown={this.handleOnKeyDown(1)}
+            type="button"
           >
-            First
-          </a>
+            {intl.formatMessage({ id: 'Terra.paginator.first' })}
+          </button>
           )
         }
-        <a
+        <button
           aria-disabled={selectedPage === 1}
-          aria-label="previous"
+          aria-label={intl.formatMessage({ id: 'Terra.paginator.previous' })}
           className={cx(['nav-link', 'left-controls', 'previous', 'icon-only', selectedPage === 1 && 'is-disabled'])}
           tabIndex={selectedPage === 1 ? null : '0'}
           onClick={this.handlePageChange(previousPageIndex)}
           onKeyDown={this.handleOnKeyDown(previousPageIndex)}
+          type="button"
         >
-          <span className={cx('visually-hidden')}>Previous</span>
+          <span className={cx('visually-hidden')}>{intl.formatMessage({ id: 'Terra.paginator.previous' })}</span>
```

Comment:
Could this be updated to use [Visually hidden text](https://engineering.cerner.com/terra-ui/#/components/terra-visually-hidden-text/visually-hidden-text/visually-hidden-text)?

Reply:
Completed. Added here https://github.com/cerner/terra-core/pull/2485/commits/a9892bd3801d58101322ebb40a768b38cd276906